third_party/terraform/tests/resource_compute_instance_migrate_test.go

```diff
@@ -22,63 +22,93 @@ func TestComputeInstanceMigrateState(t *testing.T) {
 		Attributes   map[string]string
 		Expected     map[string]string
 	}{
-		"v0.4.2 and earlier": {
-			StateVersion: 0,
-			Attributes: map[string]string{
-				"metadata.#":           "2",
-				"metadata.0.foo":       "bar",
-				"metadata.1.baz":       "qux",
-				"metadata.2.with.dots": "should.work",
-			},
-			Expected: map[string]string{
-				"metadata.foo":       "bar",
-				"metadata.baz":       "qux",
-				"metadata.with.dots": "should.work",
-			},
-		},
-		"change scope from list to set": {
-			StateVersion: 1,
-			Attributes: map[string]string{
-				"service_account.#":          "1",
-				"service_account.0.email":    "xxxxxx-compute@developer.gserviceaccount.com",
-				"service_account.0.scopes.#": "4",
-				"service_account.0.scopes.0": "https://www.googleapis.com/auth/compute",
-				"service_account.0.scopes.1": "https://www.googleapis.com/auth/datastore",
-				"service_account.0.scopes.2": "https://www.googleapis.com/auth/devstorage.full_control",
-				"service_account.0.scopes.3": "https://www.googleapis.com/auth/logging.write",
-			},
-			Expected: map[string]string{
-				"service_account.#":                   "1",
-				"service_account.0.email":             "xxxxxx-compute@developer.gserviceaccount.com",
-				"service_account.0.scopes.#":          "4",
-				"service_account.0.scopes.1693978638": "https://www.googleapis.com/auth/devstorage.full_control",
-				"service_account.0.scopes.172152165":  "https://www.googleapis.com/auth/logging.write",
-				"service_account.0.scopes.299962681":  "https://www.googleapis.com/auth/compute",
-				"service_account.0.scopes.3435931483": "https://www.googleapis.com/auth/datastore",
-			},
-		},
+		// "v0.4.2 and earlier": {
```

Comment:
If we're commenting this code, we may as well delete it. Is there a strong reason to keep it in the file instead of in source control?

Reply:
crap, forgot to un comment. thanks.